src/resource/resource.cpp

```diff
@@ -81,17 +81,24 @@ Name Name::from_string(std::string name) {
     if (!std::regex_match(name, NAME_REGEX)) {
         throw "Received invalid Name string: " + name;
     }
-    std::vector<std::string> matches;
-    boost::split(matches, name, boost::is_any_of(":"));
-
-    Subtype subtype = Subtype::from_string(matches.at(1));
-
-    std::string remote = matches.at(2);
-    if (remote.size() > 0) {
-        remote.pop_back();
+    std::vector<std::string> slash_splits;
```

Comment:
(q) just to confirm, this change was tested, yeah? I have no idea on reading if it's right or not, lol

Reply:
It was used successfully in `get_dependencies` but I hadn't actually tested it. Just did, and discovered the `remote` logic was slightly incorrect (`pop_back()` is no longer needed). The code correctly parses `namespace:type:subtype/remote:name` and `namespace:type:subtype/name`.

Reply:
Great, thanks!